src/genie/libs/parser/iosxe/show_run.py

```diff
@@ -646,8 +646,8 @@ def cli(self, interface=None, output=None):
         # switchport port-security mac-address sticky 1020.4bb1.6f2f
         p61 = re.compile(r"^switchport port-security mac-address sticky (?P<value>([a-fA-F\d]{4}\.){2}[a-fA-F\d]{4})$")
 
-        # source template USER_NoAuth
-        p62 = re.compile(r"^source template (?P<template>\w+)$")
+        # source template USER_No_802.1X_Auth
+        p62 = re.compile(r"^source template (?P<template>[a-zA-Z0-9@#$&()-`.+,/]*)$")
```

Comment:
Can you add more examples where all these special characters are needed?
A unit test that covers this change would be needed as well

Moreover, I think this pattern can be simplified. I believe that this could achieve the same results"
```suggestion
        # source template USER_NoAuth
        # source template USER_No_802.1X_Auth
        p62 = re.compile(r"^source template (?P<template>[\w#$&-`.]+)$")
```